apps/home/templates/apps_home/blocks/text_with_image_block.html

```diff
@@ -13,6 +13,11 @@ <h2>
                         {{ value.title }}
                     </h2>
                     {{ value.body|richtext }}
+                    <div>
+                        {% if value.link %}
+                        <a class="quote-ds-block__link" href="{{ value.link.url }}">{% trans 'Read more' %}&#8594;</a>
```

Comment:
The `&#8594;` is the arrow, right? We do normally use font awesome there instead. Just use arrow-right: https://fontawesome.com/v5.15/icons/arrow-right?style=solid But don't forget to add aria-hidden=trrue.

Reply:
Awesome, thanks:) I changed it, but I used the same for quote_block so should we change that or just keep in mind that we change it next time we touch it?

Reply:
Yes, please change it in both and add `aria-hidden="true"`.
(You always need to add either the `aria-hidden` or an `aria-label` to the font awesome images to make then accessible.)

Reply:
Thanks! I added it to the other font awesome images as well, and I hope it's all fine now :)